src/styles/bar.css

```diff
@@ -63,8 +70,21 @@
   text-indent: -9999em;
```

Comment:
Technically, I agree. But I probably won't rely on `.coz-hidden` as it assumes the app uses cozy-ui, which we can't expect for future contributions. The Bar must stay standalone. Whatever, I can duplicate the mixin here :+1: